airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -217,7 +228,10 @@ def get_upstream_asset_events(
 @dag_run_router.post(
     "/{dag_run_id}/clear",
     responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]),
-    dependencies=[Depends(action_logging())],
+    dependencies=[
+        Depends(requires_access_dag(method="POST", access_entity=DagAccessEntity.RUN)),
+        Depends(action_logging()),
```

Comment:
Why is that a POST ? Clearing a dagrun run mutate the run ? 

It was a PUT in the legacy implementation. (It was a POST method but a PUT permission which makes sense)

Reply:
Thanks! I've updated it.